### DISCUSSION TITLE
Navigation component

### DISCUSSION THREAD

**Discussion by maxpetts** [ariakit/ariakit#1733] [Questions] [answered]
I noticed an issue in 2018 discussing the implementation of a nav component: [#88](https://github.com/ariakit/ariakit/issues/88), and then a subsequent pull request implementing this component: [#92](https://github.com/ariakit/ariakit/pull/92). However I can't find any trace of this being used in the newer releases?

Additionally there is not a task present mentioning a nav component in the [[v2] Examples](https://github.com/ariakit/ariakit/issues/939). Has the nav component been replaced with a MenuBar?

[mdn web docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role) recommends to use the HTML5 nav element instead of roles.

In an attempt to keep the markup as semantic as possible; is it preferable to wrap a button as a link in the default React nav component? EG:
```jsx
<nav>
    <Button as="a" href="#">
        Home
    </Button>
</nav>
```

Apologies if I'm missing something, but I believe an ariakit nav component could be greatly beneficial, I am willing to work on the implementation of this. Alternatively if there is already an approach implemented, please could someone point me in the right direction - and I shall update relevant docs.

**Comment by diegohaz** [comment, marked as answer]
The Navigation component was just a `<nav>` element with some default styles back when the library still shipped with styles (v0). Since we made the library unstyled, those components were unnecessary.

If you want a navigation component now, just use `<nav>`.

`MenuBar` should be used only if your nav is designed to work like a system menu bar, which is probably not the case.

If your navigation contains dropdown menus that follow the menu patterns, use `Menu` inside it. If the menus are triggered by hover instead, use `Hovercard`, etc.
